app/controllers/dashboard_controller.rb

```diff
@@ -755,12 +755,10 @@ def tl_gen_timeline_data
   end
 
   def get_session_data
-    @layout       = "login"
-    @current_page = session[:vm_current_page] # current page number
+    @layout = "login"
   end
 
   def set_session_data
-    session[:layout]          = @layout
-    session[:vm_current_page] = @current_page
+    session[:layout] = @layout
```

Comment:
If we always set `@layout` to login, I don't see any point in saving it back to the session after...